### ISSUE TITLE
TCGAtranslateID not working?

### ISSUE THREAD

**Issue by simomounir** [waldronlab/TCGAutils#8]
Hi all,

I found your package an it happened to be useful since I want to convert some TCGA barcodes to UUID. It worked once but then it did not want to work again even if I did not change the input.

Trying to run this now:
TCGAtranslateID(TCGAbarcode("TCGA-44-6147-11A-01R-1858-07"), type="file_id")

but I get:

Warning message:
In TCGAtranslateID2(TCGAbarcode("TCGA-44-6147-11A-01R-1858-07"),  :
  No identifier data returned

What could be the problem?

Thanks

**Comment by LiNk-NY** [comment]
Hi Mohamed, @simomounir 

I've started looking into this. The GDC API is quite a challenge to figure out. 
I've looked at alternatives. For example, using the `GenomicDataCommons` package
to associate files with TCGA barcodes but it also is a bit difficult to get that type of information. 
There is no standard query to translate a UUID to a barcode since UUIDs can be assigned to files,
patients, and samples.

I will be asking the GDC package maintainer for some insights. 

Regards,
Marcel
